QuasarHelios2015/src/org/team1540/quasarhelios/Elevator.java

```diff
@@ -105,8 +105,13 @@ public static void setup() {
             return f;
         };
         
-        FloatMixing.pumpWhen(QuasarHelios.constantControl, Mixing.select((BooleanInputPoll) overrideEnabled, main, override), winch);
-
+        BooleanInput maxCurrent = BooleanMixing.createDispatch(FloatMixing.floatIsAtLeast(winchCAN.asStatus(ExtendedMotor.StatusType.OUTPUT_CURRENT), 
+                ControlInterface.mainTuning.getFloat("elevator-max-current", 40)), QuasarHelios.globalControl);
+        
+        BooleanMixing.onPress(maxCurrent).send(BooleanMixing.getSetEvent(BooleanMixing.combine(raising, lowering), false));
+        
+        FloatMixing.pumpWhen(QuasarHelios.constantControl, Mixing.select(overrideEnabled, main, override), winch);
```

Comment:
The `(BooleanInputPoll)` was there to make sure that `select`'s output will always update - not just when `overrideEnabled` changes.